### PR TITLE
#62 Test improved

### PR DESCRIPTION
**Pull request by longtimeago**

### CODE REVIEW

**Comment by longtimeago** [comment]
@karato please, find someone for review

**Comment by karato** [comment]
Thanks, let me find someone who can review this pull request

**Comment by karato** [comment]
@krzyk do this review

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by longtimeago** [comment]
@yegor256 Need your confirmation here

File: src/test/java/com/nerodesk/TkGetFileTest.java [hunk 1]
```diff
@@ -68,13 +65,18 @@ public void returnsFileContent() throws Exception {
...
-            .matcher(String.format("http://localhost:8080/api/file/%s", path));
-        matcher.find();
-        final RqRegex regex = Mockito.mock(RqRegex.class);
-        Mockito.when(regex.matcher()).thenReturn(matcher);
         MatcherAssert.assertThat(
-            IOUtils.toString(new TkGetFile(storage).route(regex).act().body()),
+            IOUtils.toString(
+                new TkGetFile(storage).route(
+                    new RqRegex.Fake(
+                        "/api/file/(?<path>[^/]+)",
```
Comment (yegor256) [inline src/test/java/com/nerodesk/TkGetFileTest.java:73]: I don't see the point of this `/api/file/` prefix here. Why can't we just use `(?<path>.*)` here?

File: src/test/java/com/nerodesk/TkGetFileTest.java [hunk 2]
```diff
@@ -68,13 +65,18 @@ public void returnsFileContent() throws Exception {
...
         final String content = "some text content";
         storage.put(path, IOUtils.toInputStream(content));
-        final Matcher matcher = Pattern.compile(TkGetFile.PATH)
-            .matcher(String.format("http://localhost:8080/api/file/%s", path));
-        matcher.find();
-        final RqRegex regex = Mockito.mock(RqRegex.class);
-        Mockito.when(regex.matcher()).thenReturn(matcher);
         MatcherAssert.assertThat(
-            IOUtils.toString(new TkGetFile(storage).route(regex).act().body()),
+            IOUtils.toString(
```
Comment (yegor256) [inline src/test/java/com/nerodesk/TkGetFileTest.java]: I would recommend to use `RsPrint` instead and then `printBody()`

**Comment by yegor256** [comment]
@longtimeago two comments above

File: src/test/java/com/nerodesk/TkGetFileTest.java (hunk 1 above)
Comment (longtimeago) [inline src/test/java/com/nerodesk/TkGetFileTest.java:73]: Do you propose to use pattern `(?<path>[^/]+)` and query `path` ?

File: src/test/java/com/nerodesk/TkGetFileTest.java (hunk 2 above)
Comment (longtimeago) [inline src/test/java/com/nerodesk/TkGetFileTest.java]: Sorry, use ```RsPrint``` instead of what?

Comment (longtimeago) [inline src/test/java/com/nerodesk/TkGetFileTest.java]: got it, thanks

**Comment by longtimeago** [comment]
outdated :(

**Comment by karato** [comment]
@krzyk I just added **16 mins** to your account, many thanks for your contribution..; review comments (c=1) added as a bonus; +16 added to your rating, current score is: [+5140](http://www.netbout.com/b/35035?open=rating)

**Pull request closed without merging**